ocfweb/api/api.py

```diff
@@ -1,18 +0,0 @@
-from django.http import JsonResponse
-from ocflib.lab.hours import Day
-
-
-def hours(request):
```

Comment:
was this file meant to be deleted?

Reply:
I was confused on that too, but I don't see it being used anywhere that I can tell, so I think it's ok?

Reply:
yeah it is deleted because we don't see it get to be used anywhere.